www/js/app.js

```diff
@@ -580,17 +580,49 @@ define(['jquery', 'zimArchiveLoader', 'util', 'uiUtil', 'cookies','abstractFiles
      * Displays the zone to select files from the archive
      */
     function displayFileSelect() {
-        $('#openLocalFiles').show();
-        $('#archiveFiles').on('change', setLocalArchiveFromFileSelect);
+        document.getElementById('openLocalFiles').style.display = 'block';
+        // Make the whole app a drop zone
+        var dropZone = document.getElementById('search-article');
+        dropZone.addEventListener('dragover', function(e) {
+            e.stopPropagation();
+            e.preventDefault();
+            e.dataTransfer.dropEffect = 'copy';
```

Comment:
Wouldn't the 'link' dropEffect be more appropriate?
Considering the size of ZIM files, a 'copy' icon made me reluctant to drop the files, thinking it would start copying a lot of GB.

Reply:
@mossroy Good idea, I'll try it out!

Reply:
Done.